### PR TITLE
Update contributing.md

### LINKED ISSUE #36: Update Contributing.md
Change contributing.md to something like this

https://github.com/arshadkazmi42/custom-date-formatting/blob/master/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Nelson-Chinedu**
Fixes #36 
update Contributing.md file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+- Create a branch using `git checkout -b feature-branch`
+- Make the required changes
+- Run below command to verify all tests are passing
+  - `npm install` (This will install all required dependencies)
+  - `npm test`
+- Create a pull request using below commands
+  - `git add --all`
+  - `git commit -m "your commit message"`
+  - `git push origin feature-branch`
+- Go to [Repository](https://github.com/arshadkazmi42/custom-date-formatting/)
```
Comment (arshadkazmi42) [inline CONTRIBUTING.md]: Can you change this url to this
`https://github.com/arshadkazmi42/simple-date-validator/`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+- Create Pull Request against `master` branch
+- Add a suitable title and description to the pull request and tag the issue number in Pull Request description, if the pull request is related to some issue logged here: [Issues](https://github.com/arshadkazmi42/custom-date-formatting/issues)
```
Comment (arshadkazmi42) [inline CONTRIBUTING.md]: Same here, change url to
`https://github.com/arshadkazmi42/simple-date-validator/issues`

**Comment by Nelson-Chinedu** [comment]
ok got that.... will fix that

**Comment by Nelson-Chinedu** [comment]
ok cool... look forward to contributing more

**Pull request merged**